Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:latest
+
+ENV config_dir=/root/.chiadog/config.yaml
+ENV TZ=UTC
```

Comment:
nitpick: inconsistent letter casing for ENV variables.

Reply:
Fix'd. This is a potentially breaking incompatibility to anyone running the original image, but since it's a manual action to switch to the official one, not a big deal.